Homework/src/io/codelex/arithmetic/practice/SumAverageRunningInt.java

```diff
@@ -0,0 +1,20 @@
+package io.codelex.arithmetic.practice;
+
+public class SumAverageRunningInt {
+    public static void main(String[] args) {
+        int sum = 0;
+        float average;
+        int lowerBound = 1;
+        int upperBound = 100;
+        int stepCounter = 0;
+
+        for (int number = lowerBound; number <= upperBound; ++number) {
+            sum += number;
+            stepCounter++;
```

Comment:
Šo drīzāk varēja matemātiski aprēķināt, ja zināms upper un lower bound nevis ciklā skatīt.